### PR TITLE
Fix broken log-level for values other than debug.

### PR DESCRIPTION
**Pull request by Harkishen-Singh**
Signed-off-by: Harkishen Singh <harkishensingh@hotmail.com>

Fixes: #557

### CODE REVIEW

File: scripts/errcheck_excludes.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 // Never check for logger errors.
-(github.com/go-kit/kit/log.Logger).Log
+(github.com/go-kit/log.Logger).Log
```
Comment (Harkishen-Singh) [inline scripts/errcheck_excludes.txt:2]: Some help here will be required. Thank you.

**Comment by sysadmind** [comment]
I think the change to the new `github.com/go-kit/kit/log` was intentional from @roidelapluie. They may be able to help understand why the log levels are broken.

**Comment by Harkishen-Singh** [comment]
@sysadmind the changes in this PR were inspired from node_exporter's way of implementing logging. That's how I got it fixed.

**Comment by roidelapluie** [comment]
github.com/go-kit/log is the new one

**Review by sysadmind — APPROVED** [review]
LGTM

**Comment by evkuzin** [comment]
@roidelapluie please approve or give an advice how to fix it another way. The whole journald is full of debug logs because of this bug.

**Review by deflinhec — APPROVED** [review]
As [go-kit#1173](https://github.com/go-kit/kit/pull/1173) mentioned

**Review by sgolod — APPROVED** [review]

**Review by ppihus — APPROVED** [review]
Done according to https://github.com/go-kit/kit/pull/1173

**Comment by ppihus** [comment]
@SuperQ, could you please approve this? Due to high amount of logs for many db clusters, we're seeing a significant increase in our logging ingestion costs.

**Comment by FlxPeters** [comment]
Any news here? We face the same problem like @ppihus. Filter logs on ingest is possible but nasty.
Would be great if this can be fixed soon as it seams we have a solution :-)

**Comment by mindw** [comment]
@wrouesnel , @SuperQ , we're seeing the same issue and had to downgrade to 0.9.x. Is there anything that can be done to move this one along?

**Comment by zawadaa** [comment]
Looks like this project is abandond :-(

File: scripts/errcheck_excludes.txt (hunk 1 above)
Comment (SuperQ) [reply scripts/errcheck_excludes.txt:2]: This is fine, it's a bit of a weird issue. The errors returned by `.Log` in this library are not useful, as it's all handled at compile time. So it the linter is miss-reporting things.

**Review by SuperQ — APPROVED** [review]
Thanks, sorry for delay in reviewing.

**Comment by SuperQ** [comment]
Sorry, I've been very busy with things and have not had the time to keep up with work on this exporter. And I really should, since my $dayjob has a bunch of PG servers.

**Comment by zawadaa** [comment]
Work can be absorbing. ;-) It's good that everything is OK with your health.
I've build on my laptop - works like a charm :-)

**Pull request merged**